app/src/app/pages/shop/shared/button/button.component.html

```diff
@@ -1,4 +1,4 @@
-<button class="product_admin">
+<button [routerLink]="['/admin']" class="product_admin">
```

Comment:
Create an @Input for this route with a value in which we will specify a reference, because this component is reusable